workspace.code-workspace

```diff
@@ -57,4 +65,4 @@
     "azurite.location": "root/.azurite",
     "azureFunctions.showExtensionsCsprojWarning": false
   }
-}
+}
```

Comment:
Newline.

Reply:
fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/20